### PR TITLE
new for configs

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

File: docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md [hunk 1]
```diff
@@ -61,7 +61,9 @@ curl 127.0.0.1:19779/flags
 
 ## Configuration files
 
-Nebula Graph provides two initial configuration files for each service, `<service_name>.conf.default` and `<service_name>.conf.production`. Users can use them in different scenarios conveniently. The default path is `/usr/local/nebula/etc/`.
+### Configuration files for clusters installed from source, with an RPM/DEB package, or a TAR package
+
+Nebula Graph provides two initial configuration files for each service, `<service_name>.conf.default` and `<service_name>.conf.production`. You can use them in different scenarios conveniently. The default path is `/usr/local/nebula/etc/` for clusters installed from source and with a RPM/DEB package; the path is `<install_path>/<tar_package_directory>/etc` for clusters installed with a TAR package.
```
Comment (izhuxiaoqing) [inline docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md:66]:
```suggestion
Nebula Graph provides two initial configuration files for each service, `<service_name>.conf.default` and `<service_name>.conf.production`. You can use them in different scenarios conveniently. For clusters installed from source and with a RPM/DEB package, the default path is `/usr/local/nebula/etc/`. For clusters installed with a TAR package, the path is `<install_path>/<tar_package_directory>/etc`.
```
we seldom use semicolon in docs as i recall

File: docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md [hunk 2]
```diff
@@ -61,7 +61,9 @@ curl 127.0.0.1:19779/flags
 
 ## Configuration files
 
-Nebula Graph provides two initial configuration files for each service, `<service_name>.conf.default` and `<service_name>.conf.production`. Users can use them in different scenarios conveniently. The default path is `/usr/local/nebula/etc/`.
+### Configuration files for clusters installed from source, with an RPM/DEB package, or a TAR package
```
Comment (izhuxiaoqing) [inline docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md:64]: what does "installed from souce" mean? the source code?

File: docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md [hunk 3]
```diff
@@ -83,9 +85,36 @@ Each initial configuration file of all services contains `local_config`. The def
 
     It is not recommended to modify the value of `local_config` to `false`. If modified, the Nebula Graph service will first read the cached configurations, which may cause configuration inconsistencies between clusters and cause unknown risks.
 
+### Configuration files for clusters installed with Docker Compose
+
+For clusters installed with Docker Compose, the configuration file's default installation path of the cluster is `<install_path>/nebula-docker-compose/docker-compose.yaml`. The parameters in the `command` field of the file are the start parameters for each service.
```
Comment (izhuxiaoqing) [inline docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md:90]: "start parameters" looks a bit strange here.

File: docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md [hunk 4]
```diff
@@ -83,9 +85,36 @@ Each initial configuration file of all services contains `local_config`. The def
...
+For clusters installed with Kubectl through Nebula Operator, the configuration file's path is the path of the cluster YAML file. You can modify the configuration of each service through the `spec.{graphd|storaged|metad}.config` parameter.  
+
+!!! note
+
+    The services cannot be configured for clusters installed with Helm.
+
 ## Modify configurations
 
-By default, each Nebula Graph service gets configurations from its configuration files. Users can modify configurations and make them valid according to the following steps:
+By default, each Nebula Graph service gets configurations from its configuration files. You can modify configurations and make them valid according to the following steps:
```
Comment (izhuxiaoqing) [inline docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md:102]:
```suggestion
By default, each Nebula Graph service gets configured from its configuration files. You can modify configurations and make them valid according to the following steps:
```

File: docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md [hunk 5]
```diff
@@ -83,9 +85,36 @@ Each initial configuration file of all services contains `local_config`. The def
...
+* For clusters installed from source, with a RPM/DEB, or a TAR package
+
+  1. Use a text editor to modify the configuration files of the target service and save the modification.
+
+  2. Choose an appropriate time to restart **all** Nebula Graph services to make the modifications valid.
+
+* For clusters installed with Docker Compose
+
+  1. In the `<install_path>/nebula-docker-compose/docker-compose.yaml` file, modify the configurations of the target service.
+  2. In the `nebula-docker-compose` directory, run the command `docker-compose up -d` to restart the configuration-changed service.
```
Comment (izhuxiaoqing) [inline docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md:113]:
```suggestion
  2. In the `nebula-docker-compose` directory, run the command `docker-compose up -d` to restart the service involving configuration modifications.
```

**Review by foesa-yang — DISMISSED** [review]

File: docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md (hunk 3 above)
Comment (abby-cyber) [reply docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md:90]: > 

has the same meaning with Launch Parameters. will use launch instead

File: docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md (hunk 2 above)
Comment (abby-cyber) [reply docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md:64]: yup, means the source code

**Review by foesa-yang — APPROVED** [review]

**Pull request merged**